modules/apps/list-type/list-type-test/src/testIntegration/java/com/liferay/list/type/service/test/ListTypeDefinitionLocalServiceTest.java

```diff
@@ -173,6 +185,36 @@ public void testUpdateListTypeDefinition() throws Exception {
 			0,
 			_listTypeEntryLocalService.getListTypeEntriesCount(
 				listTypeDefinition.getListTypeDefinitionId()));
+
+		ListTypeDefinition systemListTypeDefinition =
+			_addSystemListTypeDefinition();
+
+		externalReferenceCode = RandomTestUtil.randomString();
+
+		String liferayMode = SystemProperties.get("liferay.mode");
+
+		SystemProperties.clear("liferay.mode");
+
+		try {
+			systemListTypeDefinition =
+				_listTypeDefinitionLocalService.updateListTypeDefinition(
+					externalReferenceCode,
+					systemListTypeDefinition.getListTypeDefinitionId(),
+					TestPropsValues.getUserId(),
+					Collections.singletonMap(LocaleUtil.getDefault(), name),
+					_listTypeEntryLocalService.getListTypeEntries(
+						systemListTypeDefinition.getListTypeDefinitionId()));
+		}
+		finally {
+			SystemProperties.set("liferay.mode", liferayMode);
+		}
+
+		Assert.assertNotEquals(
+			externalReferenceCode,
+			systemListTypeDefinition.getExternalReferenceCode());
```

Comment:
Is it not supposed to edit the `externalReferenceCode`?

Reply:
That's right. Currently, you can only edit the name.